scio-macros/src/main/scala/com/spotify/scio/MagnoliaMacros.scala

```diff
@@ -34,13 +34,7 @@ private[scio] object MagnoliaMacros {
       )
     }
 
-    val magTree = magnolia.Magnolia.gen[T](c)
-
-    def getLazyVal =
-      magTree match {
-        case q"val $name = $body; $rest" =>
-          body
-      }
+    val getLazyVal = magnolia.Magnolia.gen[T](c)
```

Comment:
🎨 should we rename this ... or even move it to `removeAnnotations.transform( ... )`?